things/index.html

```diff
@@ -1,3 +1,4 @@
+<!-- having two pages named index would be confusing for other programmers I think, consider renaming your portfolio page -->
```

Comment:
It wouldn't be confusing because each of my index.html's with the exception of the landing page, is inside of a folder. This removes the .html from the URL per one of our previous assignments instructions.